### DISCUSSION TITLE
Reusing your custom onnx

### DISCUSSION THREAD

**Discussion by Schlockschlock** [QIN2DIM/hcaptcha-challenger#97] [Q&A]
Hello,

I'm starting to learn Python and I'm trying to do myself a hcaptcha solver like you, for training. For the "new" challenges (bridge, rooms, horses, etc.) I understand that you're using custom trained models (bridge.onnx, lions.onnx, etc.). So I would like to reuse those custom ONNX file for the specific challenges. That's why I'm trying to read your code to find the 2 most important parts: 
- where you're loading those ONNX custom model files
- and where you're performing the object detection with the boxes coordinates

As far as I understand, those 2 functions are located in the /src/services/hcaptcha_challenger/solutions/yolo.py right?
Thank you

**Comment by QIN2DIM** [comment]
> where you're loading those ONNX custom model files

- YOLO: https://github.com/QIN2DIM/hcaptcha-challenger/blob/59ff1af90a8ac1e1a1ad96bdcad16982048cc9b2/src/services/hcaptcha_challenger/solutions/yolo.py#L135

- ResNet: https://github.com/QIN2DIM/hcaptcha-challenger/blob/59ff1af90a8ac1e1a1ad96bdcad16982048cc9b2/src/services/hcaptcha_challenger/solutions/resnet.py#L41

> where you're performing the object detection with the boxes coordinates

- YOLO: https://github.com/QIN2DIM/hcaptcha-challenger/blob/59ff1af90a8ac1e1a1ad96bdcad16982048cc9b2/src/services/hcaptcha_challenger/solutions/yolo.py#L189

- ResNet: https://github.com/QIN2DIM/hcaptcha-challenger/blob/59ff1af90a8ac1e1a1ad96bdcad16982048cc9b2/src/services/hcaptcha_challenger/solutions/resnet.py#L95

**Comment by Schlockschlock** [comment]
Thank you very much

**Reply by QIN2DIM** [reply]
🥂
